src/aws/osml/model_runner/tile_worker/tile_worker.py

```diff
@@ -100,10 +103,15 @@ def process_tile(self, image_info: Dict, metrics: MetricsLogger = None) -> None:
                 if len(features) > 0:
                     self.feature_table.add_features(features)
 
+                self.region_request_table.add_tile(
+                    image_info.get("image_id"), image_info.get("region_id"), image_info.get("region"), TileState.SUCCEEDED
+                )
         except Exception as e:
-            logging.error("Failed to process region tile!")
-            logging.exception(e)
-            self.feature_detector.error_count += 1  # borrow the feature detector error count to tally other errors
+            self.failed_tile_count += 1
+            logging.error(f"Failed to process region tile with error: {e.with_traceback}")
```

Comment:
Wouldn't this be: `e.with_tracekback()` because you would be calling a method.

Reply:
Updated